README.md

```diff
@@ -1,2 +1,120 @@
-# mailgun-relayery
-A Go server providing an additional security layer for using the MailGun API
+Mailgun Relayery
+================
+
+Mailgun Relayery is a tool providing an additional security layer for the MailGun API.
```

Comment:
"Mailgun Relayery gives you relaying with different API keys through the single-key MailGun API."

There is actually no extra security involved -- or at least, it's hard to understand it from the single phrase.